Classes/Driver/AmazonS3Driver.php

```diff
@@ -778,7 +817,7 @@ public function copyFolderWithinStorage($sourceFolderIdentifier, $targetFolderId
      * @return string The file contents
      * @throws InvalidPathException
      */
-    public function getFileContents($fileIdentifier)
+    public function getFileContents($fileIdentifier): string
```

Comment:
De gereturnde `file_get_contents` geeft `false` terug indien iets niet opgehaald kan worden. Zelfde geldt voor `file_put_contents` hieronder

Reply:
Aangepast door `return file_get_contents() ?: ''` te gebruiken in `getFileContents`.

In `setFileContents` is er een exception toegevoegd op het moment dat `file_put_contents` false als waarde teruggeeft.